components/claim/components/email_to_oauth.tsx

```diff
@@ -11,27 +10,41 @@ import * as Utils from 'utils/utils.jsx';
 import {t} from 'utils/i18n.jsx';
 import LoginMfa from 'components/login/login_mfa.jsx';
 import LocalizedInput from 'components/localized_input/localized_input';
+import { boolean } from '@storybook/addon-knobs';
 
-export default class EmailToOAuth extends React.PureComponent {
-    static propTypes = {
-        newType: PropTypes.string,
-        email: PropTypes.string,
-        siteName: PropTypes.string,
-    };
+type Props = {
+    newType?: any,
+    email?: any,
+    siteName?: string,
```

Comment:
do we really need any?

this happens across the PR, can you take a look at the others as well?

Reply:
change it to string too like the above that you commented?

Reply:
It is either `string` or whatever the right type is. `any` should probably just be reserved to when we really don't know what will be for the variable, otherwise we are not getting any benefits from typing.